lute/cli/commands/transform/lib/arguments.luau

```diff
@@ -0,0 +1,69 @@
+local strings = require("./strings")
+
+export type Config = {
+	--- Prevent making changes to files
+	dryRun: boolean,
+	--- Path to the migration module to execute
+	migrationPath: string,
+	--- Configuration flags to pass to the migration
+	migrationOptions: { [string]: string },
+	--- List of file paths to process
+	--- Note: no directory traversal or filtering has been applied on this list
+	filePaths: { string },
+}
+
+local function parse(arguments: { string }): Config
+	-- TODO: replace with proper CLI system
```

Comment:
We have argparsing working rn - probably worth re-writing in terms of that.

Reply:
retracting this comment - cli parsing is currently not in std, and so we can't use it for now.